tests/src/Kernel/SmsFrameworkProviderTest.php

```diff
@@ -383,6 +388,7 @@ public function testEventsQueueIncoming() {
     $this->container->get('cron')->run();
 
     $expected[] = SmsEvents::MESSAGE_INCOMING_PRE_PROCESS;
+    $expected[] = 'incomingEvent'; // Plugin incoming event.
```

Comment:
I'm not saying the `$expected[] = 'incomingEvent';` assertion is aesthetically good. But if we could refer to method names in PHP like we can with classes ($class::class - class name resoution) then I would do that here.